src/Vulkan/RHI/VkTexture.cpp

```diff
@@ -1,15 +1,16 @@
-﻿#include "VkTexture.h"
+﻿#include "VkAllocator.h"
 
 #include <Vex/Bindings.h>
 #include <Vex/RHIBindings.h>
 
+#include <Vulkan/RHI/VkAllocator.h>
 #include <Vulkan/RHI/VkCommandPool.h>
 #include <Vulkan/RHI/VkDescriptorPool.h>
+#include <Vulkan/RHI/VkTexture.h>
```

Comment:
Same as buffer, some include incorrectness with "VkAllocator" being the main include.